lib/Command/Updater/Job.php

```diff
@@ -64,12 +73,27 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $output->writeln("Checking update Status");
         $date = new DateTime();
         $date->setTimestamp($this->statusService->getUpdateTime());
-        $output->writeln("Last Execution was ".$date->format('Y-m-d H:i:s e'));
+        $now = new DateTime('now');
+        $elapsedInterval = $now->diff($date);
+        $output->writeln("Last Execution was ".$date->format('Y-m-d H:i:s e').
+          $elapsedInterval->format("; %h hours, %i minutes, %s seconds ago"));
```

Comment:
I just added the relative time display. At least for me, it's more relevant how long ago a last update was executed, than at what exact time it was. Can be taken out however, my main goal is the feedback through the exit code, so that scripts can act on it.

Reply:
Problem is that people might already have this scripted.

Reply:
Another option could be adding a new command-line switch, which triggers this check, and the exit code. Thus no existing script would possibly break.

Reply:
I added a --check-elapsed option to the command. Currently, if both are specified, the --reset takes precedence. I'm thinking either erroring out instead - or, better yet, maybe in that case the job could be automatically reset _if_ it was executed too long ago, along with displaying the message. The docs might warn that this has its own risks, so should be used with caution.

Reply:
So, is this good as is, or I should do either of the above suggested changes?